src/components/tools/Settings/SelectStation.svelte

```diff
@@ -8,6 +8,7 @@
 
   export let selectedId;
   export let items;
+  export let title = "SELECT STATION";
```

Comment:
Just noting that the default `title` prop here is all caps but in others it's title case. I think for a11y it's generally better to use normal casing for text and transform the text using CSS for visual display.

Reply:
Got it. Fixed via https://github.com/berkeley-gif/caladapt-website-2021/commit/97dc0ad6c103ebda2a133563f5018ab95a4d7bfc